### PR TITLE
47 - adding docker-compose for running tests locally

### PR DESCRIPTION
**Pull request by ginettev**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+FROM adoptopenjdk/openjdk11:alpine-slim
```
Comment (SrinivasanTarget) [inline Dockerfile:1]: Adopt JDK series is deprecated. Can we use debug distroless images from Google?

Comment (ginettev) [reply Dockerfile:1]: Ok. Will check out the correct version to use here.

Comment (ginettev) [reply Dockerfile:1]: the issue is gcr.io/distroless/java:11-debug is not containing /bin/sh and docker is throwing errors while trying to start the service

File: docker-compose-test.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+#
+version: "3.8"
+services:
+  # Basic development container.
+#  dev:
+#    # user: "${UID}:${GID}"
+#    image: gcr.io/distroless/java:11-debug-nonroot
+#    volumes:
+#      - .:/code
+#    entrypoint: [ "tail", "-f", "/etc/hosts" ]
```
Comment (SrinivasanTarget) [inline docker-compose-test.yaml:15]: Shall we remove the bunch of commented lines? otherwise LGTM

**Review by SrinivasanTarget — CHANGES_REQUESTED** [review]

File: docker-compose-test.yaml (hunk 2 above)
Comment (SrinivasanTarget) [reply docker-compose-test.yaml:15]: @KeerthanaR29 ping

Comment (KeerthanaR29) [reply docker-compose-test.yaml:15]: we need line 4 and 5 for command to execute that docker compose test.yaml..But we can remove other commands

Comment (KeerthanaR29) [reply docker-compose-test.yaml:15]: we need 10 - 15 lines.I forgot to uncomment that

**Review by SrinivasanTarget — APPROVED** [review]

**Pull request merged**